release/release_logs/2.0.0/stress_tests/stress_test_many_tasks.json

```diff
@@ -3,45 +3,45 @@
         {
             "perf_metric_name": "stage_0_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 6.4732019901275635
+            "perf_metric_value": 6.37376856803894
         },
         {
             "perf_metric_name": "stage_1_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 20.34087462425232
+            "perf_metric_value": 23.254157304763794
         },
         {
             "perf_metric_name": "stage_2_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 53.219086503982545
+            "perf_metric_value": 71.0486969947815
         },
         {
             "perf_metric_name": "stage_3_creation_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 0.05356264114379883
+            "perf_metric_value": 0.054947853088378906
         },
         {
             "perf_metric_name": "stage_3_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 2350.621628522873
+            "perf_metric_value": 2473.47390460968
         },
         {
             "perf_metric_name": "stage_4_spread",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 3.2340626645789263
+            "perf_metric_value": 2.0478648041499694
         }
     ],
-    "stage_0_time": 6.4732019901275635,
-    "stage_1_avg_iteration_time": 20.34087462425232,
-    "stage_1_max_iteration_time": 21.677543878555298,
-    "stage_1_min_iteration_time": 19.695393323898315,
-    "stage_1_time": 203.40884971618652,
-    "stage_2_avg_iteration_time": 53.219086503982545,
-    "stage_2_max_iteration_time": 54.085320711135864,
-    "stage_2_min_iteration_time": 51.03796076774597,
-    "stage_2_time": 266.0965692996979,
-    "stage_3_creation_time": 0.05356264114379883,
-    "stage_3_time": 2350.621628522873,
-    "stage_4_spread": 3.2340626645789263,
+    "stage_0_time": 6.37376856803894,
+    "stage_1_avg_iteration_time": 23.254157304763794,
+    "stage_1_max_iteration_time": 23.794161319732666,
+    "stage_1_min_iteration_time": 22.56113624572754,
+    "stage_1_time": 232.54165983200073,
+    "stage_2_avg_iteration_time": 71.0486969947815,
+    "stage_2_max_iteration_time": 72.61798572540283,
+    "stage_2_min_iteration_time": 70.24156618118286,
+    "stage_2_time": 355.2453670501709,
+    "stage_3_creation_time": 0.054947853088378906,
+    "stage_3_time": 2473.47390460968,
+    "stage_4_spread": 2.0478648041499694,
```

Comment:
The high signal indictor is `stage_3_time` was at 3k level. https://b534fd88.us1a.app.preset.io/superset/explore/p/Zxv7VWrA4qR/